crates/primitives/src/bytecode.rs

```diff
@@ -44,16 +47,18 @@ pub struct Bytecode {
 }
 
 impl Default for Bytecode {
+    #[inline]
     fn default() -> Self {
         Bytecode::new()
     }
 }
 
 impl Bytecode {
-    /// Create [`Bytecode`] with one STOP opcode.
+    /// Creates a new [`Bytecode`] with exactly one STOP opcode.
+    #[inline]
```

Comment:
The compiler is smart and in most cases will do a good job on the inline front, so adding `#[inline]` everywhere is just noisy.

Reply:
This is not the case for public functions, which will not get inlined across crate boundaries without LTO: <https://nnethercote.github.io/perf-book/inlining.html>

Also `#[inline]` is just a hint, and given that these functions are very trivial I don't see why not to hint to the compiler that they should be inline (like `Vec::len`).

I've had this discussion before, I don't want to argue about this again so if you're really keen on this I'll just remove all of the hints.

Reply:
I was thinking just about in crate use, I tried to play with it with the opcode function and would have better results when I just leave it to the compiler. Let's leave it with `#[inline]` as it makes sense for `Bytecode`.